packages/ui/v2/core/Meta.tsx

```diff
@@ -41,16 +41,16 @@ export function MetaProvider({ children }: { children: React.ReactNode }) {
  * elsewhere (ie. on a Heading, Title, Subtitle, etc.)
  * @example <Meta title="Password" description="Manage settings for your account passwords" />
  */
-export default function Meta({ title, description, backButton }: MetaType) {
+export default function Meta({ title, description, backButton, CTA }: MetaType) {
   const { setMeta, meta } = useMeta();
 
   /* @TODO: maybe find a way to have this data on first render to prevent flicker */
   useEffect(() => {
     if (meta.title !== title || meta.description !== description) {
-      setMeta({ title, description, backButton });
+      setMeta({ title, description, backButton, CTA });
```

Comment:
When the `<Meta />` component is rendered it will set `MetaContext` for that page. Line 24 (can't comment) is where the `useMeta` hook is declared. This hook returns `MetaContext`